proving_system/stark/src/air/example/cairo.rs

```diff
@@ -0,0 +1,305 @@
+use lambdaworks_math::field::{
```

Comment:
I would name this module `cairo` instead of `example`

Reply:
I agree, lets merge this now and do that later 👍